core/src/test/java/org/infinispan/distribution/DisabledL1Test.java

```diff
@@ -22,11 +22,11 @@ public void testRemoveFromNonOwner() {
       for (Cache<Object, String> c : caches) assert c.isEmpty();
```

Comment:
I know this is not the purpose of your PR, but this test seems to me unnecessary as it is, unless it depends on DistSyncFuncTest instead of BaseDistFunctionalTest